src/pathfora.js

```diff
@@ -142,7 +141,7 @@
   // NOTE HTML templates
   // FUTURE Move to separate files and concat
   /* eslint-disable indent */
-  var templates = {{templates}};
+  var templates = '{{templates}}';
```

Comment:
One idea: we can use a syntactic macro. Instead of having handlebars, we'd just call a function
```javascript
var templates = HBS('templates');
```
```javascript
function HBS() {
  return ...; // Maybe this returns fixture data, or it might just be a noop
}
```
but at compile time we could inline the actual templates and eliminate the `HBS` function using dead code removal